content-metadata-protobuf/compiled/proto/Channel_pb.d.ts

```diff
@@ -0,0 +1,45 @@
+// package: 
```

Comment:
Should we include the compiled outputs in the repo, seems inadvisable?

Reply:
I'm making an exception here, because this is a consumable package and it seems a burden to expect the consumer to also have the protocol buffer compiler to use the package.